sdk/python/feast/infra/registry/contrib/azure/azure_registry_store.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import os
+import uuid
+from datetime import datetime
+from pathlib import Path
+from tempfile import TemporaryFile
+from urllib.parse import urlparse
+
+from feast.infra.registry.registry import RegistryConfig
+from feast.infra.registry.registry_store import RegistryStore
+from feast.protos.feast.core.Registry_pb2 import Registry as RegistryProto
+
+REGISTRY_SCHEMA_VERSION = "1"
+
+
+class AzBlobRegistryStore(RegistryStore):
+    def __init__(self, registry_config: RegistryConfig, repo_path: Path):
+        try:
+            import logging
+
+            from azure.identity import DefaultAzureCredential
```

Comment:
can you generally verify that a normal pip install without the azure extra works correctly? in case we accidentally pull in a deep

Reply:
Yup I just did a normal install in a new environment and it works fine.